cmd/pocket/main.go

```diff
@@ -0,0 +1,16 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+)
+
+var version string
+
+func main() {
+	v := flag.Bool("version", false, "")
```

Comment:
I'm a bit confused by what this is trying to do. The passed in version is a bool and we're printing an unassigned version string?

Don't we want the flag to write the value in &version?

Reply:
The variable is set via a linker flag (see `magefile.go`). This is something I need to make explicit as part of the documentation that Luis required of me 😄

Reply:
Made also add a comment in the code about the variable?